packages/rich-text/src/component/index.native.js

```diff
@@ -1051,6 +1061,9 @@ export class RichText extends Component {
 			textDecorationColor: defaultTextDecorationColor,
 			fontFamily: defaultFontFamily,
 		} = getStylesFromColorScheme( styles.richText, styles.richTextDark );
+		const textLinkColor = this.getLinkTextColor(
```

Comment:
Nitpick: Rename the variable to match the `RCTAztecView` prop.
```suggestion
		const linkTextColor = this.getLinkTextColor(
```

**NOTE:** If this change is applied, keep in mind that it also requires updating line 1162.

Reply:
Thank you, updated in https://github.com/WordPress/gutenberg/pull/38474/commits/6b4d8ff1c5bae95539fed8cd2651f511d37decde